renovate.json

```diff
@@ -53,6 +53,20 @@
       "groupName": "typescript-eslint",
       "matchPackagePatterns": "@typescript-eslint/*"
     },
+    {
+      "groupName": "ESLint plugins",
+      "matchPackageNames": [
```

Comment:
I didn't want to use `eslint-plugin-*` to avoid combining the bumps of https://github.com/mui/mui-x/blob/a198b26371fa8ed6185175426e983efffd2b1e3a/package.json#L152 as it can often have lots of unsolvable problems given the state of the plugin.

Reply:
What about enabling `eslint-plugin-*` and just skip `eslint-plugin-react-compiler` instead

Reply:
Is there a way to do it for a group config? 🤔

Reply:
We use the following rule [in Toolpad](https://github.com/mui/mui-toolpad/blob/c8e05a9385434c0526fca1125e7e071cc7a7a7b7/renovate.json#L41-L45)
```tsx
    {
      "groupName": "eslint",
      "matchPackageNames": ["eslint"],
      "matchPackagePatterns": ["^eslint-", "^@eslint/", "^@eslint-", "^@typescript-eslint/"]
    },
```

Reply:
@LukasTy you can negate pattern like `!eslint-plugin-react-compiler `

Reply:
And how do you deal with dependency bumping when `eslint-plugin-react-compiler` changes are very time consuming?
I would imagine that other ESLint dep bumping might be stuck for a while. 🙈

Reply:
Well, let's test the negating. 🤞

Reply:
> And how do you deal with dependency bumping when eslint-plugin-react-compiler changes are very time consuming?
> I would imagine that other ESLint dep bumping might be stuck for a while. 🙈

What do you mean?

This config is just for grouping, meaning `eslint-plugin-react-compiler` will still bump alone.

Reply:
> This config is just for grouping, meaning `eslint-plugin-react-compiler` will still bump alone.

I meant the Toolpad config. 😉 
In their case, it looks like the `react-compiler` plugin would be included in the same group...

Reply:
Negating didn't work, even though it is documented 🤔 
https://docs.renovatebot.com/configuration-options/#matchpackagenames

Reply:
yeah... 🙈 😢 
That's what I was afraid of. 😆

Reply:
Looks like my bad: https://github.com/mui/mui-x/pull/14577 🙈 🤦